src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/CreateConnectionRequestCommand.php

```diff
@@ -77,4 +77,15 @@ public function getApplicant(): Contact
     {
         return $this->applicant;
     }
+
+    private function getUniqueConnectionRequests(array $connectionRequests): array
+    {
+        $result = [];
+        foreach ($connectionRequests as $key => $connectionRequest) {
+            if (!in_array($connectionRequest->institution, $result, true)) {
```

Comment:
Personally I'm a fan of using `array_key_exists` for checking uniqueness in a collection. Where I use the key of the array to identify the content associated to it. :man_shrugging:. This works equally well. 

(Just to let you know, not to nudge you into changing this)